src/unlimited.cpp

```diff
@@ -429,10 +408,6 @@ void UnlimitedSetup(void)
         mallocedStats.push_front(new CStatHistory<uint64_t>("net/send/msg/" + *i));
     }
 
-    xpeditedBlk.reserve(256);
```

Comment:
I know why reserve() exists, and that's exactly what we want -- a single allocation.  Not for performance optimization but for heap stability.

If you read my comments above carefully, I am not saying that we should remove the lock.  I am saying that it is good that the lock is probably not necessary at the machine level since this will increase program stability even if bugs exist.